templates/zerver/help/message-a-stream-by-email.md

```diff
@@ -15,3 +15,5 @@ contents.
 
 4. After you send an email to the stream's email address, your email will be
 forwarded to the stream and sent as a message.
+
+Note: Sending message to a stream by email is currently not configured on [zulip](chat.zulip.org).
```

Comment:
Changed to say something like this, using the `!!! tip` feature that we use in a lot of our /help/ docs.
```
This feature requires server-level configuration in the EMAIL GATEWAY INTEGRATION section of  /etc/zulip/settings.py. On a server that hasn't configured this feature, the Email address section will not be displayed.
```